src/base/views/mouse-tool.ts

```diff
@@ -79,6 +84,32 @@ export class MouseTool implements IVNodePostprocessor {
         }
     }
 
+    protected handleEvents<K extends keyof MouseListener>(methodName: K, models: SModelRoot[], events: MouseEvent[]) {
```

Comment:
To prevent code duplication, the handleEvent function should now call this handleEvents function with single-element arrays.

Reply:
This seems a good change.